tests/test_export_db.py

```diff
@@ -0,0 +1,229 @@
+###############################################################################
+#
+# MIT License
+#
+# Copyright (c) 2023 Advanced Micro Devices, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+###############################################################################
+
+import os
+import sys
+import argparse
+import logging
+import pytest
+
+from sqlalchemy.orm import Query
+from sqlalchemy.ext.declarative import declarative_base
+
+sys.path.append("../tuna")
+sys.path.append("tuna")
+
+this_path = os.path.dirname(__file__)
+
+from tuna.miopen.subcmd.export_db import (
+    arg_export_db, get_filename, get_base_query, get_fdb_query, get_pdb_query,
+    get_fdb_alg_lists, build_miopen_fdb, write_fdb, export_fdb,
+    build_miopen_kdb, write_kdb, export_kdb, create_sqlite_tables, get_cfg_dict)
+from tuna.utils.db_utility import DB_Type
+from tuna.miopen.db.tables import MIOpenDBTables, ConfigType
+from tuna.dbBase.sql_alchemy import DbSession
+from tuna.miopen.db.find_db import ConvolutionFindDB
+from utils import add_test_session, DummyArgs
+
+
+@pytest.fixture
+def mock_args():
```

Comment:
@reidkwja  we already have something like this in /test/utils.py. If these are used for go_fish you can use the GoFishArgs